scripts/components/FlAccountUser.ts

```diff
@@ -57,15 +57,15 @@ export default class FlAccountUser extends FlAccountUserDesign {
     }
     set onProfileClick(value: () => Promise<any>) {
         this.__onProfileClick = value;
-        this.lblAccountEditIcon.onTouchEnded = (isInside) => isInside && value();
+        this.lblEditIcon.onTouchEnded = (isInside) => isInside && value();
     }
     get onPhotoClick(): () => Promise<any> {
         return this.__onPhotoClick;
     }
     set onPhotoClick(value: () => Promise<any>) {
         if (value) {
             this.__onPhotoClick = value;
-            this.lblAccountEditIcon.onTouchEnded = (isInside) => isInside && value();
+            this.lblEditIcon.onTouchEnded = (isInside) => isInside && value();
```

Comment:
We should move this event to constructor like

this.lblEditIcon.on(View.Events.TouchEnded, () => {
    this._value && this._value();
});
@suleymanbasbug

Reply:
ok